README.md

```diff
@@ -1,2 +1,8 @@
-# kics-github-action-demo
-A demo repo to show KICS Github Action in Action
+# 🤖 KICS GitHub Actions Demo
+
+This repository is a simple repository to show how KICS GitHub Action can be set.
```

Comment:
Propose:
This repository is a simple repository to show how KICS GitHub Action can be set.
-->
This repository shows how KICS GitHub Action can be set.